tracpro/settings/base.py

```diff
@@ -271,6 +271,7 @@ def _org_scheduler_task(task_name):
     "Administrators": (
         'orgs.org_home',
         'orgs.org_edit',
+        'orgs.fetch_runs',
```

Comment:
That's the way of assigning permissions to groups - adding them to PERMISSIONS is the way of adding them to the database